### DISCUSSION TITLE
Error preventing websites from loading in Chromium Edge

### DISCUSSION THREAD

**Discussion by richsnyder0** [darkreader/darkreader#11433] [Q&A]
Long-time user, love the product!

Over the past couple days, some websites have failed to load while DarkReader is running. Turning off the extension allows the sites to load. Here is the error message I can see in the Network tab for one of the sites that's trying to load. I did not have any trouble with loading the websites until this week.

ERROR TypeError: Cannot use 'in' operator to search for 'insertRule' in undefined
    at Function.value (libs.22765.min.js:46:29495)
    at libs.22765.min.js:46:19285
    at Object.invoke (main-es2015.e73853d1fb2e386ff2dc.js:1:1011768)
    at main-es2015.e73853d1fb2e386ff2dc.js:1:1009827
    at C (main-es2015.e73853d1fb2e386ff2dc.js:1:987480)
    at Ke (main-es2015.e73853d1fb2e386ff2dc.js:1:1009805)
    at l (main-es2015.e73853d1fb2e386ff2dc.js:1:995692)
    at Tt (main-es2015.e73853d1fb2e386ff2dc.js:1:996007)
    at a.bootstrap (libs.22765.min.js:48:14053)
    at main-es2015.e73853d1fb2e386ff2dc.js:1:3352367

**Comment by bershanskiy** [comment]
Thanks for the bug report. The stack trace does not look like something Dark Reader ships. Perhaps, there is an odd interaction between Dark Reader and some website code? Could you please provide the following: (1) example website where this happens and (2) if you disable all other extensions besides Dark Reader, does the bug still occur?

**Comment by richsnyder0** [comment]
Looks like every time I get an Azure SSO prompt the browser hangs and that error message comes up. I turned off my other extensions and with only Dark Reader on, it occurs.
